dm/worker/worker.go

```diff
@@ -330,11 +333,16 @@ func (w *Worker) resetSubtaskStage(etcdCli *clientv3.Client) (int64, error) {
 			// TODO: right operation sequences may get error when we get etcdErrCompact, need to handle it later
 			// For example, Expect: Running -(pause)-> Paused -(resume)-> Running
 			// we get an etcd compact error at the first running. If we try to "resume" it now, we will get an error
-			err = w.operateSubTaskStage(stage, subtaskCfg)
+			op, err := w.operateSubTaskStage(stage, subtaskCfg)
```

Comment:
how about return the operate type string for these functions, so don't need to judge whether `op` is nil.